internal/cloud/backend.go

```diff
@@ -366,7 +366,7 @@ func (b *Cloud) setConfigurationFields(obj cty.Value) tfdiags.Diagnostics {
 	// or using TF_ORGANIZATION. Since PrepareConfig() validates that one of these
```

Comment:
This says TF_ORGANIZATION, but I found a bunch of other places where the old symbols exist in comments and e2e tests.